Documents/LearnJs/Constructor(prototype)/js/script.js

```diff
@@ -0,0 +1,94 @@
+// //1
+function Car(make, model, year, color, price) {
+    this.make = make;
+    this.model = model;
+    this.year = year;
+    this.color = color;
+    this.price = price;
+}
+
+function Garage() {
+    this.cars = [];
+    this.addCar = function (make, model, year, color, price) {
+        const newCar = new Car(make, model, year, color, price);
+        this.cars.push(newCar);
+        return "New car was added: " + make + " " + model;
+    }
+
+    this.addCarFromInput = function () {
+        const make = document.getElementById("makeInput").value;
+        const model = document.getElementById("modelInput").value;
+        const year = parseInt(document.getElementById("yearInput").value);
+        const color = document.getElementById("colorInput").value;
+        const price = parseFloat(document.getElementById("priceInput").value);
+
+        const message = this.addCar(make, model, year, color, price);
+        document.querySelectorAll('input[type="text"], input[type="number"]').forEach(input => input.value = '');
+        displayCars();
+
+        return message;
+    }
+}
+
+const myGarage = new Garage();
+
+function displayCars() {
+    const carList = document.getElementById("carList");
+    carList.innerHTML = "";
+
+    myGarage.cars.forEach(function (car, index) {
+        const listItem = document.createElement("li");
```

Comment:
This function should be moved from forEach to a separate method.

Reply:
const createCarListItem = function(car, index) {
        const listItem = document.createElement("li");